### PR TITLE
Signup functionality with ui

### PR DESCRIPTION
**Pull request by Eldho1234**
Created signup page (UI only)

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -7,9 +8,9 @@ void main() {
 class MyApp extends StatelessWidget{
   @override
   Widget build(BuildContext context) {
-   return const MaterialApp(
+   return MaterialApp(
     debugShowCheckedModeBanner: false,
-    home: Text("hello"),
+    home: SignUpPage(),
```
Comment (Eldho1234) [inline lib/main.dart:13]: Add scaffold

**Review by Eldho1234 — COMMENTED** [review]
Reviewed

File: lib/view/sing_up.dart [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+                  )),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+bool isEmailVAlid(String email) {
```
Comment (vishnu303) [inline lib/view/sing_up.dart:229]: Test comment

**Review by vishnu303 — APPROVED** [review]
reviewed

**Pull request merged**